app/vmselect/promql/eval.go

```diff
@@ -364,6 +364,10 @@ func evalAggrFunc(qt *querytracer.Tracer, ec *EvalConfig, ae *metricsql.AggrFunc
 		args: args,
 		ec:   ec,
 	}
+
+	qtChild := qt.NewChild("eval %s", ae.Name)
+	defer func() { qtChild.Done() }()
```

Comment:
This can be simplified to `defer qtChild.Done()`. Additionally, it looks like it is better from readability and maintainability PoV to call `qtChild.Done()` just after the `af(afa)` call on the next line.